frontend/src/feature/category/model/category.ts

```diff
@@ -2,4 +2,6 @@ export type Category = {
   id: string,
   name: string,
   access: string,
+  numberOfCards?: number,
```

Comment:
Should we add this information to the CategoryDTO?

Reply:
we could of course - and I thought about it. I guess there are Pros and Cons. But I am open for both.